deploy.sh

```diff
@@ -6,6 +6,9 @@ tmux_layouts_dir="$HOME/.tmuxinator"
 vim_config_dir="$HOME/.vim"
 vim_config_path="$HOME/.vimrc"
 zsh_config_path="$HOME/.zshrc"
+if [[ $1 = '--bypass-yn-prompt' ]]; then
```

Comment:
This is awesome! However, in the event we decide to extend custom functionality and add other flags this won't be immediately extendable. 

Totally non-blocking, but might I suggest adapting this first parameter check into a loop which runs through all parsed command line args?

```
while [[ "$1" != "" ]]; do
    case $1 in
      --bypass-yn-prompt) shift
                          bypass_yn_prompt="${bypass_yn_prompt:-true}"
                          ;;
      *)				  exit 1   # invalid arg
    esac
    shift
  done
```

Note this extension `"${ENVIRONMENT_VARIABLE:-VALUE}"` is nice because it respects configuration that is passed in via environment variable 😄.

Reply:
This could also be placed into the `main` function, since the scope of this variable is global if we felt it appropriate.